Realm/RealmSwift/Aliases.swift

```diff
@@ -20,5 +20,11 @@ import Realm
 
 // These types don't change when wrapping in Swift
 // so we just typealias them to remove the 'RLM' prefix
+
+// MARK: Aliases
+
+/// An enum of the different property types supported by Realm.
```

Comment:
Should list the values in the docs unless Jazzy handles that automatically somehow.